default/package.json

```
@@ -10,21 +10,21 @@
   "dependencies": {
     "@react-router/node": "*",
     "@react-router/serve": "*",
-    "isbot": "^5.1.17",
+    "isbot": "^5.1.23",
```

Let's keep the other dependencies as-is and only do `vite` for now

```suggestion
    "isbot": "^5.1.17",
```